infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -169,6 +183,13 @@ func NewTFBlueprintTest(t testing.TB, opts ...tftOption) *TFBlueprintTest {
 			tft.setupDir = setupDir
 		}
 	}
+	// Common retryable errors
+	if tft.retryableTerraformErrors == nil {
+		tft.retryableTerraformErrors = map[string]string{}
+	}
+	for k, v := range commonRetryableErrors {
+		tft.retryableTerraformErrors[k] = v
+	}
```

Comment:
I am leaning towards first adding just the option. We can add this by default later if we see these commonly.

Reply:
removed